client/components/banner/style.scss

```
@@ -122,10 +122,12 @@
 .banner__icon-plan {
 	display: flex;
 	margin-right: 16px;
```

This should be enough.

```suggestion
	margin-right: 16px;
	flex-shrink: 0;
```

---

You're right! thanks for that I won't need to update the width to `max-width` but what do you think about keeping `flex-direction:column` and `justify-content:center` to center the icon vertically?